### PR TITLE
[UA] Credits

### PR DESCRIPTION
**Pull request by qwrtln**

### CODE REVIEW

**Review by SerhiiSUr — APPROVED** [review]

File: translations/credits.tex/ua.po [hunk 1]
```diff
@@ -132,12 +147,14 @@ msgid ""
...
 msgstr ""
+"\\textbf{Перекладацький інжиніринг}: Віктор Марчук, Анджей Вьонцек\n"
+"\n"
 
 #. type: multicols*
 #: sections/credits.tex:25
 #, no-wrap
 msgid ""
-"\\phantom{Translators placeholder}\n"
+"\\textbf{Переклад}: Сергій Сергеєв\n"
```
Comment (qwrtln) [inline translations/credits.tex/ua.po:157]: Mention that it's Ukrainian.

File: translations/credits.tex/ua.po [hunk 2]
```diff
@@ -108,6 +117,8 @@ msgid ""
 "\\textbf{Layout design}: Joris de Kleer, Kate Katrankova, Andrzej Wiącek\n"
 "\n"
 msgstr ""
+"\\textbf{Дизайн макета}: Йоріс де Клер, Катерина Катранкова, Анджей Вьонцек\n"
```
Comment (qwrtln) [inline translations/credits.tex/ua.po:120]:
```suggestion
"\\textbf{Дизайн макета}: Йоріс де Клер, Кейт Катранкова, Анджей Вьонцек\n"
```

**Comment by qwrtln** [comment]
po4a check

File: translations/credits.tex/ua.po [hunk 3]
```diff
@@ -154,6 +154,9 @@ msgstr ""
 #: sections/credits.tex:25
 #, no-wrap
 msgid ""
+"\\phantom{Translators placeholder}\n"
+"\n"
+msgstr ""
 "\\textbf{Переклад}: Сергій Сергеєв\n"
```
Comment (qwrtln) [inline translations/credits.tex/ua.po:160]: Don't shy away from your last name, and please say in what language you translated.

File: translations/credits.tex/ua.po [hunk 4]
```diff
@@ -140,6 +157,9 @@ msgid ""
 "\\phantom{Translators placeholder}\n"
 "\n"
 msgstr ""
+"\\textbf{Переклад українською}: Сергій Сергеєв\n"
+"\n"
+msgstr ""
```
Comment (qwrtln) [inline translations/credits.tex/ua.po:162]:
```suggestion
```

**Pull request merged**